Autopilot/CMakeLists.txt

```diff
@@ -120,6 +120,7 @@ elseif(${KIND_OF_BUILD} STREQUAL "UNIT_TESTS")
     ${CMAKE_CURRENT_SOURCE_DIR}/PathManager/Inc
     ${CMAKE_CURRENT_SOURCE_DIR}/PathManager/Inc/cruisingMode
     ${CMAKE_CURRENT_SOURCE_DIR}/PathManager/Inc/landingMode
+    ${CMAKE_CURRENT_SOURCE_DIR}/PathManager/Inc/takeoffMode
     ${CMAKE_CURRENT_SOURCE_DIR}/TelemetryManager/Inc
```

Comment:
Run `./Tools/build.bash -c`. It will fail because you forgot to add: `PathManager/Inc/takeoffMode` in the `include_directories` on line 26